public/index.html

```diff
@@ -1,11 +1,15 @@
 <!doctype html>
-<html>
+<html itemscope itemtype="http://schema.org/Organization">
 <head>
   <meta charset="utf-8">
   <meta name="viewport" content="width=device-width, minimum-scale=1.0, initial-scale=1.0, user-scalable=yes">
   <meta name="apple-mobile-web-app-capable" content="yes">
   <meta name="apple-mobile-web-app-status-bar-style" content="black">
 
+  <meta itemprop="name" content="Hundred Zeros">
+  <meta itemprop="description" content="Home of the Hundred Zeros Destiny clan. PvE and PvP, Xbox and Playstation: we cover them all! Special emphasis on fun and camaraderie">
+  <meta itemprop="image" content="images/snippet.png">
```

Comment:
No, I shrunk it for efficiency. I also like picking names that reflect the role of the asset rather than its content. It makes replacements easier.